{
  if (node.getContainingElement(PyElementTypes.LOOPS) == null) {
    getHolder().createErrorAnnotation(node,"'continue' outside of loop");
    return;
  }
  PyTryExceptStatement tryStatement=node.getContainingElement(PyTryExceptStatement.class);
  if (tryStatement != null) {
    PsiElement parent=node.getParent();
    while (parent != null) {
      if (parent == tryStatement.getFinallyStatementList()) {
        getHolder().createErrorAnnotation(node,"'continue' not supported inside 'finally' clause");
        break;
      }
      parent=parent.getParent();
    }
  }
}
